versions/run.ver

```diff
@@ -1,4 +1,6 @@
 export evs_ver=v1.0
+export met_ver=11.0.0
+export metplus_ver=5.0.0
 
```

Comment:
@MarcelCaron-NOAA -- any reason whey these old versions were introduced?

Reply:
No these were not intentional additions.  Removing them--thanks.